### PR TITLE
Added glossary show api endpoint [#181798616]

### PR DESCRIPTION
**Pull request by dougmartin**

### CODE REVIEW

File: spec/controllers/api/v1/glossary_controller_spec.rb [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+    it "recognizes and generates #show" do
+      expect({:get => "api/v1/glossaries/#{glossary.id}.json"}).to route_to(
+        :controller => "api/v1/glossaries",
+        :action => "show",
+        :id => "#{glossary.id}",
+        :format => "json"
+      )
+    end
+
+    it "when user is anonymous, shows an activity's full json" do
```
Comment (scytacki) [inline spec/controllers/api/v1/glossary_controller_spec.rb:62]:
```suggestion
    it "when user is anonymous, shows an glossary's full json" do
```

File: spec/controllers/api/v1/glossary_controller_spec.rb [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+      json_response = JSON.parse(response.body)
+      expect(json_response).to eq({
+        id: glossary.id,
+        name: glossary.name,
+        user_id: glossary.user_id,
+        json: JSON.parse(stringifed_json)
+      }.as_json)
+    end
+
+    it "when user is anonymous, shows an activity's json contents" do
```
Comment (scytacki) [inline spec/controllers/api/v1/glossary_controller_spec.rb:75]:
```suggestion
    it "when user is anonymous, shows an glossary's json contents" do
```

**Review by scytacki — APPROVED** [review]
Just one suggestion on switching "activity" to "glossary".
Otherwise looks good.

File: spec/controllers/api/v1/glossary_controller_spec.rb (hunk 1 above)
Comment (dougmartin) [reply spec/controllers/api/v1/glossary_controller_spec.rb]: Fixed in https://github.com/concord-consortium/lara/pull/920/commits/60aa5f3d7999183b56375fa2fec4f44d9c4e6480

File: spec/controllers/api/v1/glossary_controller_spec.rb (hunk 2 above)
Comment (dougmartin) [reply spec/controllers/api/v1/glossary_controller_spec.rb]: Fixed in https://github.com/concord-consortium/lara/pull/920/commits/60aa5f3d7999183b56375fa2fec4f44d9c4e6480

**Pull request merged**